pkg/scm/gitlab/context.go

```diff
@@ -52,6 +52,12 @@ func NewContext(ctx context.Context, baseURL, token string) (*Context, error) {
 	evalContext.MergeRequest = evalContext.Project.MergeRequest
 	evalContext.Project.MergeRequest = nil
 
+	evalContext.MergeRequest.Assignees = evalContext.MergeRequest.CurrentAssignees.Nodes
+	evalContext.MergeRequest.CurrentAssignees = nil
+
+	evalContext.MergeRequest.Reviewers = evalContext.MergeRequest.CurrentReviewers.Nodes
+	evalContext.MergeRequest.CurrentReviewers = nil
```

Comment:
have you checked if those are `[]` when empty or `nil` ? the API is all over the place with these lol

If there are no assignees, are `evalContext.MergeRequest.CurrentReviewers` then `[]` or `nil` 🤔

Reply:
i checked and they are [], also are [] as part of the graphql schema

Reply:
cool!